pyproject.toml

```diff
@@ -1,15 +1,58 @@
 [build-system]
+build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools >= 40.0",
+  "setuptools",
+  "setuptools-scm",
```

Comment:
```suggestion
```

Let's remove this, we still have the hardcoded version below, and it's better to do a one-to-one migration here.